### DISCUSSION TITLE
video has 23.976 progressive and 29.97 hard telecined video. ripbot detelecines this .mkv video fine

### DISCUSSION THREAD

**Discussion by jimbobub** [staxrip/staxrip#1313] [Q&A]
The .mkv video was ripped from a dvd using makemkv. It contains a vob segment of 23.976 and 29.97 hard telecined vobs. Staxrip is showing the average frame rate of the whole video around 26fps in the target window with no ivtc engaged. I tried all ivtc options and 25fps(animel ivtc) is the closet to come to 23.976 cfr x264 which is my goal. Decomb Ivtc  and tivtc show around 21fps. Ripbot has worked fine but it does not have ac3 passthrough so I converted the audio from aac to ac3 for blu-ray compatibility. I did figure out how to pad the video to 1280 x 720 to maintain the 4:3 a.r. Any help would be appreciated, thanks.

**Comment by jessielw** [comment]
StaxRip supports QTGMC, this is slower but superior to other deinterlacing methods.

**Comment by Dendraspis** [comment]
You could also use filters like `AssumeFPS()`, `ChangeFPS()` and/or `ConvertFPS()` to change the frame rate.
